src/components/Slider.react.js

```diff
@@ -133,6 +139,31 @@ Slider.propTypes = {
      */
     max: PropTypes.number,
 
+    tooltip: PropTypes.exact({
+       /**
+        * Determines whether tooltips should always be visible
+        * (as opposed to the default, visible on hover)
+        */
+        visible: PropTypes.bool,
```

Comment:
`visible` seems like a fuzzy name for hover vs. always visible

Reply:
Yeah, I agree. I left it like that because that's the [API underneath](https://github.com/react-component/tooltip#props) (see "tooltip"). But considering our discussion on not needing a 1-1 API match for DCC, if you'd rather have an alias i.e. `visible` = `always_visible` I'll add it.

Reply:
I like `always_visible: bool` actually - you'd only use it when it's `true` since `false` is the default. And even if you did say `always_visible: false`, it's pretty clear that that's not the same thing as "never visible", which is the implication of `visible: false`.

Reply:
OK, I've made the change.

Reply:
LGTM - @Marc-Andre-Rivet are you comfortable with this?

Reply:
Yup. All good.